src/Log/README.md

```diff
@@ -0,0 +1,116 @@
+# Log
+
+The Log package is a [psr/log](https://github.com/php-fig/log) compatible logger implementation using WordPress actions to log messages using [Monolog](https://github.com/Seldaek/monolog) behind the scenes.
+
+As this uses WordPress actions, it doesn't couple any logger to your code, and simply won't fire any log
+messages if no logger is configured.
+
+## Quick Start
+
+Register `Log_Definer.php` and `Log_Subscriber.php` in Square One Core.php. Call the Log Actions below
+in your code.
+
+## Log Action Definition
+
+```php
+/**
+ * @param string $logger_hook Use on of the constants defined in Log_Actions.
+ * @param string $message The message to log.
+ * @param mixed[] $additional_data_to_log Additional data to log. 
+ */
+do_action( string $logger_hook, string $message, array $additional_data_to_log );
+```
+
+## Example Usage
+
+```php
+do_action( \Tribe\Libs\Log\Log_Actions::INFO, 'Starting connection...' );
```

Comment:
Does Log "action" make sense here since this is a "log level"? Kinda confusing swapping of words.

Reply:
This is a log action, aka https://github.com/moderntribe/tribe-libs/pull/95/files#diff-9437c961423ea88688f4c9e87658251c3818bc352ebfcbde8bf5944c7eafde24R14. 

https://github.com/moderntribe/tribe-libs/pull/95/files#diff-9437c961423ea88688f4c9e87658251c3818bc352ebfcbde8bf5944c7eafde24R45-R52

This is a WordPress hook, not a Log Level.